T4XJYT_LGI301/T4XJYT_LGI301/Core/API/ApiDataProvider.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace T4XJYT_LGI301.Core.API
+{
+    public class ApiDataProvider : IApiDataProvider
+    {
+        
+        private readonly string _apiURL = "https://baconipsum.com/api/?type=meat-and-filler";
```

Comment:
API url-t nem célszerű hardcodeolni, helyette egy config fájlból szokás felolvasni. Jelen esetben ez nem probléma, de readonly string helyett ez akár lehetett volna const is.